### ISSUE TITLE
Embedding Elementor Template Post in WP Default Editor

### ISSUE THREAD

**Issue by RealBenihime** [pojome/elementor#5915]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Please see last 2 steps to reproduce below.

## Steps to reproduce
-Create Post Template in Elementor, apply it to all Posts
-Create a Post
-Copy the link to the Post
-Paste it on the WP Default Editor of a page
-A normal post embeds as (Title, Featured Image, Description/Except, WP icon Sitename, Share Button)
-However a Post with Elementor Template, embeds as a full webpage OR fails to embeds completely

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | Apache |  
MySQL version: | (Ubuntu) v5.5.61 |  
PHP Version: | 5.5.9-1ubuntu4.26 |  
PHP Max Input Vars: | 1000 |  
PHP Max Post Size: | 8M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected |  

</details>

**Comment by arielk** [comment]
Fixed in Pro v2.1.11

Thanks for your feedback!

**Comment by arielk** [comment]
Fixed in Pro v2.1.11

Thanks for your feedback!

**Comment by RealBenihime** [comment]
You guys rock!

I confirm the issue has been resolved.

Thanks @arielk
